llvm/lib/Transforms/Vectorize/VPlan.cpp

```diff
@@ -1072,9 +1072,13 @@ InstructionCost VPlan::cost(ElementCount VF, VPCostContext &Ctx) {
   // blocks, like the preheader or middle blocks.
   InstructionCost Cost = getVectorLoopRegion()->cost(VF, Ctx);
```

Comment:
Better bail out first if invalid?

Reply:
Yep, added a check to the if and updated the comment, thanks!